book/website/reproducible-research/rdm/personal.md

```diff
@@ -0,0 +1,138 @@
+For example, can data be de-identified or anonymised? 
+* If data cannot be made unidentifiable then you must ensure you have authority to transfer the personal data, and that there are appropriate safeguards in place to protect the data before, during and after transit.
+* Often your university or institute will provide solutions for secure file transfer. 
+Contact you research data, privacy or IT support teams for support. 
+
+**4. Encryption** 
+* Encryption provides protection by ensuring that only someone with the relevant encryption key (password) will be able to access the contents.
+    * Protect on disk level: Bitlocker
+    * Protect on “container” level (a folder containing multiple files):  Veracrypt
+    * Portable storage: Bitlocker
+    * File level / Exchange information:
+    * Simple method: use 7zip, and pack with a password
+    * More complicated to setup: use PGP tooling (can also be used to securely send mail)
+
+**5. Access permissions**
+* management of shared folder permissions
+* password protection
+
+
+**6. Anonymisation**
+
+Anonymisation is a process by which identifying information in a dataset is removed. 
+It is used primarily to enable data to be shared or published without revealing the confidential information it contains, while limiting the loss of information.
+* Where possible, direct identifiers (such as names, addresses, telephone numbers and account numbers) should be removed as soon as the identifying information is no longer needed. 
+You can delete the data or replace it with pseudonyms. 
+For qualitative data you should replace or generalise identifying characteristics when transcribing interviews.
+* De-identified data that can be re-identified using a linkage file (for example, information linking data subjects to identifiable individuals) is known as pseudonymised data. 
+NOTE: In this instance, the linkage file should be encrypted and stored securely and separately from the de-identified research data.
+  * Identification of individuals in pseudonymised or de-identified data may still be possible using combinations of indirect identifiers (such as age, education, employment, geographic area, and medical conditions). 
+Further, data and outputs containing small cell counts may be potentially disclosive, particularly where samples are drawn from small populations or include cases with extreme values or relatively rare characteristics.
+   * As such, when intending to share potentially identifiable data or the outputs generated from the data, you may need to consider more advanced anonymisation techniques such as statistical disclosure control (SDC, see [this handbook](https://securedatagroup.org/sdc-handbook/) for more information).
+* For more information about anonymisation you can watch [this webinar by Enrico Glerean on the 'Basics of anonymization'](https://www.youtube.com/watch?v=ILXeA4fx3cI)
```

Comment:
Maybe the link can stop at "this webinar by Enrico Glerean" to avoid the use of the -yze/-ization variant of the spelling? :P

Reply:
Good point, the topic is already indicated before anyway :D